test/IRGen/constant_struct_with_padding.sil

```diff
@@ -0,0 +1,26 @@
+// RUN: %swift -module-name main %s -emit-ir -o - | %FileCheck %s
```

Comment:
Are you expecting it to not be general enough already? It seems like there's nothing that's platform specific, but I don't know much about the various permutations we have, yet.